src/integration-tests/test_reconfigure_domains.py

```diff
@@ -84,7 +88,7 @@ def test_reconfigure_domain_message_limits(self, multi_node: Cluster):
         assert not self.post_n_msgs(URI_PRIORITY_2, 1)
 
         # Modify the domain configuration to hold 2 more messages.
```

Comment:
Is this comment not correct?
```suggestion
        # Modify the domain configuration to hold 10 more messages.
```